WORKSPACE

```diff
@@ -56,6 +39,9 @@ r_repository_list(
     name = "r_repositories_bzl",
     package_list = "//:r_packages.csv",
     remote_repos = {
+        "BioCsoft": "https://bioconductor.org/packages/3.11/bioc",
+        "BioCann": "https://bioconductor.org/packages/3.11/data/annotation",
+        "BioCexp": "https://bioconductor.org/packages/3.11/data/experiment",
```

Comment:
I don't think we have these 3 packages. Are they necessary? I don't see them used

Reply:
I removed them